Sources/SwiftSyntaxBuilder/gyb_helpers/ExpressibleAsConformances.py

```diff
@@ -38,7 +38,7 @@
     'StmtBuildable': [
         'CodeBlockItem'
     ],
-    'TokenSyntax': [
+    'Token': [
```

Comment:
Do we  actually use these? I think we specify them manually in `SwiftSyntaxBuilder/Token.swift`, do you think it would be worth generating `Token.swift` just to keep these conformances in sync?

Reply:
Now that I generate Token.swift again, it’s being used now :wink